### PR TITLE
Gidk 1489 chat icon

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 1]
```diff
@@ -9450,6 +9450,12 @@ span#HW_badge_cont:nth-child(3) {
         }
     }
 }
+#rcw-p91-widget-button{
+    z-index: 99;
+}
+.toast {
+ text-align: left !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:9457]: format css

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.scss [hunk 2]
```diff
@@ -25,7 +25,9 @@
 .ledger-panel table tbody tr td a {
     color: #333;
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.scss:26]: Use color variable in this

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: formatted

File: apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.scss:26]: I have not added this color right now, please focus on red and green line changes

**Review by ravinderwalkover — APPROVED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

**Review by rohit-walkover — APPROVED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**